src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.Streaming.cs

```diff
@@ -19,9 +19,11 @@ internal partial class EndpointHtmlRenderer
     private TextWriter? _streamingUpdatesWriter;
     private HashSet<int>? _visitedComponentIdsInCurrentStreamingBatch;
     private string? _ssrFramingCommentMarkup;
+    private bool _isHandlingErrors;
 
-    public void InitializeStreamingRenderingFraming(HttpContext httpContext)
+    public void InitializeStreamingRenderingFraming(HttpContext httpContext, bool isErrorHandler)
     {
+        _isHandlingErrors = isErrorHandler;
```

Comment:
Could have created a separate method for this, but I think it's fine if we leave it like this for now and clean it up later.

Reply:
Either's fine. I thought it was pretty cool that, by adding it as a param here, you forced us to account for it in both RazorComponentEndpointInvoker and RazorComponentResultExecutor (or whatever they are both called). Would have been easy to forget RazorComponentResultExecutor otherwise